apps/dashboard/src/hooks/use-create-integration.ts

```diff
@@ -0,0 +1,18 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
```

Comment:
Not related to this file, but please check the console for warnings and errors:
![Screenshot 2024-12-17 at 18 01 03](https://github.com/user-attachments/assets/bcb88b60-1a22-4f16-bec0-a5a62cfee94f)